src/client/sandbox/event/simulator.ts

```diff
@@ -610,6 +610,12 @@ export default class EventSimulator {
             });
         }
 
+        if (timeStamp && !browserUtils.isIE) {
```

Comment:
I guess this condition should be changed to `if (timeStamp && !browserUtils.isIE11) {`
because MS Edge supports the `timestamp` property (https://developer.mozilla.org/ru/docs/Web/API/Event/timeStamp)
Also, an appropriate test should be changed.

Reply:
All the browsers support the `timestamp` property. However, it is different. In MS browsers, the `timestamp` of the `mouseup` and the `click` events can be not equal to each other. However, in Chrome/Safari/FF the `timestamp` properties of these two events are the same. Thus the `browserUtils.isIE` condition is required for both IE and Edge.

Reply:
Ok